files/default/bin/github_comment.rb

```diff
@@ -0,0 +1,11 @@
+#!/opt/chef/embedded/bin/ruby
+require_relative '../lib/github_comment.rb'
+
+# quit unless we get three command line arguments
+unless ARGV.length == 3
+  puts "Requires three arguments.\n"
+  puts "Usage: gh_comments.rb <repo_name> <issue_id> <comment_text>\n"
```

Comment:
The name here doesn't match what's used. I.e. it should say ``Usage: github_comment.rb``.